### DISCUSSION TITLE
able to pick up space separated suggestion

### DISCUSSION THREAD

**Discussion by xerosanyam** [remirror/remirror#817] [Q&A]
using prosemirror-suggest,
the objective is:

We need to show mentions popup once user types any character. ie. Trigger on Free flowing text
List of options can have whitespace separated items
Activation character should not go into "start" mode again once it sees space
For example, we have few users ["Mark Zuckerberg", "Steve Jobs", "Steve Wozniak"]
User types "Steve j", then only Steve jobs should appear.

What I've tried doing is,
I made the activation character to be /\S/
This triggers our suggestions
But once it reaches space after the 1st word it re-triggers & goes into start mode. ie, query becomes empty & I am able to see all the options

In short, our list of options contain whitespace seperated items & we need suggester to not get into "start" mode once it sees space.

related to #353
